### PR TITLE
[NOTEST] Allow filtering streams in PRT runs

### PR DESCRIPTION
**Pull request by vrutkovs**
Add '{{ streams: downstream-5.5, upstream }}' to PR's body

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5b0de59a9cbf177d91782a5b2aa558e03f83e356:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5a4c8a3d75133914beabdb40c4d3404355040eff:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./scripts/dockerbot/dockerbot.py:10:1: F401 're' imported but unused
./scripts/dockerbot/dockerbot.py:13:1: F401 'yaml' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3852d58df07af467e8c1959b9ae489eb1ae72ff2:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by psav** [comment]
Command line doesn't require adding prt meta data as it gets it from the PR itself, maybe we solve this by simply doing a check for pr_metadata. If it's None, then goto github, else, use what is already given.

**Comment by vrutkovs** [comment]
@psav latest commit should handle this - pr metadata will be fetched if its missing

**Comment by dajoRH** [comment]
Lint report for commit 2f4dcf47d150ade90b3a9af7e49527615cd41d54:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./scripts/dockerbot/dockerbot.py:185:24: F821 undefined name 're'
./scripts/dockerbot/dockerbot.py:189:25: F821 undefined name 'yaml'
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0730249cfe8344c6be89d76d49f2233202d8895e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: scripts/dockerbot/check_prs.py [hunk 1]
```diff
@@ -281,6 +292,15 @@ def check_status(pr):
         pass
 
 
+def get_pr_metadata(pr_body):
```
Comment (psav) [inline scripts/dockerbot/check_prs.py]: Is it worth extracting the guts of this to a new function that both can use? putting it in dockerbot module and importing it into the check_prs?

Comment (vrutkovs) [inline scripts/dockerbot/check_prs.py]: Yeah, lets try importing

**Comment by dajoRH** [comment]
Lint report for commit 1ed2ee6888b7fe367b1091ae5f5944825b6688bc:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./scripts/dockerbot/dockerbot.py:14:1: F401 're' imported but unused
./scripts/dockerbot/dockerbot.py:15:1: F401 'yaml' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f68f933cc8a8d0a4fc4b43eccc9eff0a984e0466:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 216d34d6c16c8c1219464490e72a93e1cd5d8809:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: scripts/dockerbot/dockerbot.py [hunk 2]
```diff
@@ -499,6 +496,9 @@ def handle_output(self):
     repo.add_argument('--pr',
                       help='A PR Number (overides --branch)',
                       default=None)
+    repo.add_argument('--pr-metadata',
+                      help='A PR Number (overides --branch)',
```
Comment (psav) [inline scripts/dockerbot/dockerbot.py]: @vrutkovs can we fix the help message?

**Pull request merged**